setup.py

```diff
@@ -7,7 +7,7 @@
 
 from setuptools import setup, find_packages
 
-version = '0.0.22'
+version = '0.1.0'
```

Comment:
I think we can continue increasing the version to 0.0.23 until we feel this is a nice release version.